pages/badges/index.tsx

```diff
@@ -0,0 +1,45 @@
+import type { NextPage } from 'next'
+
+import { Layout } from '@/components/layouts/layout'
+import { SimpleGrid, Button } from '@chakra-ui/react'
+import { useNetwork } from 'wagmi'
+import { getContractAddress } from '@/utils/contractAddress'
+import { useBadges } from '@/hooks/badge/useBadges'
+import { ethers } from 'ethers'
+import { useEffect, useState } from 'react'
+import { BadgeBox } from '@/components/badges/BadgeBox'
+
+const Badges: NextPage = () => {
+  const { activeChain } = useNetwork()
+  const henkakuBadge = getContractAddress({
+    name: 'henkakuBadge',
+    chainId: activeChain?.id
+  })
+  const { badges } = useBadges(henkakuBadge)
+  const [badgeList, setBadgeList] = useState<[]>([])
```

Comment:
result of badges will be list of badge so useBadges's badges should have a type

Reply:
fixed https://github.com/henkaku-center/omise-interface/pull/102/commits/14a82c98e114ff5383b63346a1754a795600e753

However, since the contract returned an array, it could not handle objects, so type checking was performed on tuples.